stdlib/Random/docs/src/index.md

```diff
@@ -66,6 +71,18 @@ Random.MersenneTwister
 Random.RandomDevice
 ```
 
+## Cryptographically secure random numbers
+
+The Julia standard library does not provide a native cryptographically secure random number generator. For most applications,
```

Comment:
By "native", you mean written in Julia? I find it's easy to interpret this sentence as "you can't get CSPRNG" from the `Random` module, which seems to be contradicted in the next sentence...